manifests/filter/open_tracing.pp

```diff
@@ -0,0 +1,170 @@
+# == Resource:  repose::filter::open_tracing
+#
+# This is a resource for enabling emitting traces to an OpenTracing tracer (current supports Jaeger).
+# Only usable in >= 8.8.3.0
+#
+# === Parameters
+#
+# [*ensure*]
+# Bool. Ensure config file present/absent
+# Defaults to <tt>present</tt>
+#
+# [*filename*]
+# String. Config filename.
+# Defaults to <tt>open-tracing.cfg.xml</tt>
+#
+# [*service_name*]
+# String. Application name
+# Defaults to <tt>repose</tt>
+#
+# [*http_connection_endpoint*]
+# String. Set to http endpoint to push traces to.
+# Defaults to <tt>http://localhost:12682/api/traces</tt>
```

Comment:
You've still got a default here. if i want to go to udp, this value will still be set without me explicitly making it undef which is ugly in usage. it also forced a specific order to your if logic below, it's probably hinky in the template as well. Force them to make a conscious choice on which exfil strategy they are using and where it's going.